html-forms/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+      <form action="" method="POST">
+        <div>
+          <label for="email">Email</label>
+          <br>
+          <input type="email" id="email">
+        </div>
```

Comment:
We want to avoid using `<br>` tags, it is better to use `<div>`s instead. For example:

```HTML
<div>
  <label for="email">Email</label>
  <div>
    <input type="email" id="email">
  </div>
</div>
```